plugins/win-wasapi/win-wasapi.cpp

```diff
@@ -169,23 +169,31 @@ inline void WASAPISource::Start()
 
 inline void WASAPISource::Stop()
 {
+	blog(LOG_INFO, "[WASAPISource::Stop][%08X] Device '%s' Stop called", this,
+		device_id.c_str());
+
 	SetEvent(stopSignal);
 
-	if (active) {
-		blog(LOG_INFO, "[WASAPISource::Stop][%08X] Device '%s' Terminated", this,
+	if (active || captureThread.Valid()) {
```

Comment:
should we do it like this: `if (captureThread.Valid() && active) {` instead? To make sure it's valid?

Reply:
Technically only check for a valid thread is enough here. As we only wait for a thread to stop after that. 

Checking for active in addition to a thread validity does nothing here right now. 
There is a slight chance that it could be useful if future merge add something what actually depends on a active status instead of just waiting for thread to stop.